kmesh.dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openeuler/openeuler:23.09
```

Comment:
what's the different with build/docker/kmesh.dockerfile?

Reply:
IIUC, we can remove that one

Reply:
> what's the different with build/docker/kmesh.dockerfile?

we do not need  build/docker/kmesh.dockerfile, I have removed it.